src/opcodes.c

```diff
@@ -203,6 +213,17 @@ void rorOpcode(unsigned int address, struct CPU *cpu, struct BUS *bus)
   writeOnMemoryOrAccumulator(address, result, cpu, bus);
 }
 
+void rtiOpcode(unsigned int address, struct CPU *cpu, struct BUS *bus)
+{
+  char setBreakFlagFalse = ~0x10;
+  unsigned char flagsValue = popStack(cpu, bus);
+  setStatusByChar(flagsValue & setBreakFlagFalse, cpu);
+
+  unsigned char lo = popStack(cpu, bus);
+  unsigned char hi = popStack(cpu, bus);
+  cpu->pc = (hi << 8) | lo;
```

Comment:
That's amazing!
I was trying to figure this thing out and you did it!!
Awesome!